src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,27 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_CHARACTERS = 6;
```

Comment:
```suggestion
    private static final int MIN_PASSWORD_CHARACTERS = 6;
```
I suggest more informative name